Client/Application/ClientPreferences.swift

```diff
@@ -250,6 +250,8 @@ extension Preferences {
         static let listViewSide = Option<String>(key: "playlist.listViewSide", default: PlayListSide.left.rawValue)
         /// Whether to show Add to playlist Toast
         static let showToastForAdd = Option<Bool>(key: "playlist.showToastForAdd", default: true)
+        /// Whether to show the Add to Playlist URL-Bar onboarding
+        static let showAddToPlaylistURLBarOnboarding = Option<Int>(key: "playlist.showAddToPlaylistURLBarOnboarding", default: 0)
```

Comment:
Since this is a some kind of `count` preference i would rename it,
`showAddToPlaylistURLBarOnboarding` assumes it is a bool, update the comment too